queue/abstract.lua

```diff
@@ -63,6 +63,26 @@ queue.driver = {
     limfifottl  = require('queue.abstract.driver.limfifottl')
 }
 
+local function tube_release_all_tasks(tube)
+    local prefix = ('queue: [tube "%s"] '):format(tube.name)
+
+    -- We lean on stable iterators in this function.
+    -- https://github.com/tarantool/tarantool/issues/1796
+    if not qc.check_version({1, 7, 5}) then
+        log.error(prefix .. 'no stable iterator support: skip task releasing')
+        log.error(prefix .. 'some tasks may stuck in taken state perpetually')
+        log.error(prefix .. 'update tarantool to >= 1.7.5 or take the risk')
+    end
+
+    log.info(prefix .. 'releasing all taken task (may take a while)')
```

Comment:
In my opinion,  it's quick action and log is useless.

Reply:
I'll ask an order for some hot instance from our operation team.

Reply:
I got the answer that they have small amount of taken tasks (on utubettl driver).

However I suspect that different cases are possible. Don't sure here, to be honest.

Feel free to remove the print.